File: Programmers/sumDivisor/SumDivisor.java
import java.lang.Math;

class SumDivisor {
	public int sumDivisor(int num) {
		int answer = 0;
		for(int i=1; i<=Math.sqrt(num);++i)
		{
			if(num % i == 0)
			{
				if(i == num/i)
				{
					answer += i;
				}
				else
				{
					answer += i + num/i;
				}
			}
		}
		return answer;
	}

	public static void main(String[] args) {
		SumDivisor c = new SumDivisor();
		System.out.println(c.sumDivisor(9));
	}
}

File: Programmers/Level1/sumDivisor(jptjs)/sumDivisor.js
function sumDivisor(num) {
  var i = 1;
  var answer=0;
	while(i<Math.sqrt(num))
  {
    if(!(num % i))
    {
      answer += i + num/i;
    }
    ++i;
  }
	return answer;
}

// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log(sumDivisor(12));